src/relay/transforms/simplify_expr.h

```diff
@@ -54,14 +54,16 @@ class DFPatternRewrite {
       Map<DFPattern, Array<Expr>> node_map = args[2];
       *rv = this->Callback(pre, post, node_map);
     };
-    return DFPatternCallback(pattern_, PackedFunc(func), require_type_);
+    return DFPatternCallback(pattern_, PackedFunc(func), require_type_, rewrite_once_);
   }
 
  protected:
   /*! \brief The pattern for matching and rewriting. */
   DFPattern pattern_;
   /*! \brief Whether or not the rewrite requires types to be inferred. */
   bool require_type_ = true;
+  /*! \brief If True, rewrite only once. */
+  bool rewrite_once_ = false;
```

Comment:
Seems no way to change this value? And in which case we may need rewrite once in SimplifyExpr?